.github/workflows/release-generate-ci-template.yaml

```diff
@@ -85,8 +85,20 @@ jobs:
         run: |
           set -x
           git remote add fork "https://github.com/serverless-qe/release.git"
-          git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/release.git" sync-serverless-ci:sync-serverless-ci -f
-          gh pr create --base master --head serverless-qe:sync-serverless-ci --fill-verbose || true
+          branch="sync-serverless-ci"
+          remote_exists=$(git ls-remote --heads fork "$branch")
+          if [ -z "$remote_exists" ]; then
+            # remote doesn't exist.
+            git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/release.git" "$branch:$branch" -f
+            exit $?
+          fi
+          git fetch fork "$branch"
```

Comment:
Should this be in the `else` branch? Not sure what happens if the remote branch doesn't exist. We probalby suppose it always exists?